rbi/core/file.rbi

```diff
@@ -74,6 +74,11 @@ class File < IO
   extend T::Generic
   Elem = type_member(:out, fixed: String)
 
+  # most File methods take a Pathname, or a String that represents a file path
+  TFilePath = T.type_alias { T.any(String, Pathname) }
```

Comment:
We optimize `RBI`s for readability and thus try to not introduce type aliases.
Also, because this would now look like it will allow to use `File::TFilePath` in runtime that actually doesn't exist.
Please, expand these type aliases

Reply:
updated in https://github.com/sorbet/sorbet/pull/2476/commits/f13cfece58121d29a3ef08f046a6f919f466eaaa